.stylelintrc

```diff
@@ -11,6 +11,7 @@ rules:
   declaration-empty-line-before: null
   indentation: 4
   no-descending-specificity: null
+  no-duplicate-selectors: null
```

Comment:
Also please do not disable this rule

Reply:
Issue is we have a color rule above:

```css
/* Tab color tweaks */
.ui.tabular.menu .item {
    color: rgba(0, 0, 0, .5);
}
```

which then duplicates with the height rule:

```
.ui.tabular.menu {
    min-height: 36px;
}
```

I think it's fine to keep them apart because to keep the rules logically grouped but that rule prevents such grouping.

Reply:
https://stylelint.io/user-guide/ignore-code

Reply:
I think they should be moved under single group `/* Tab tweaks */`

Reply:
Merged the rules but that did destroy some git history along with it because of that linter rule.